tests/gold_tests/bad_http_fmt/bad_http_fmt.test.py

```diff
@@ -166,10 +215,3 @@ def server_cmd(resp_id):
 tr.Processes.Default.Command = "grep -e '^===' -e '^HTTP/' -e 'X-Resp-Id:' -e '<HTML>' client.log"
 tr.Processes.Default.Streams.stdout = 'client.gold'
 tr.Processes.Default.ReturnCode = 0
-
-tr = Test.AddTestRun()
-tr.Processes.Default.Command = "grep -e 'X-Req-Id:' -e 'HTTP/' -e 'Content-' server1.log"
-for n in range(2, 5):
-    tr.Processes.Default.Command += f" ; grep -e 'X-Req-Id:' -e 'HTTP/' -e '[Cc]ontent-' server{n}.log"
-tr.Processes.Default.Streams.stdout = 'server.gold'
-tr.Processes.Default.ReturnCode = 0
```

Comment:
Hmm looks like no checks of server output?  Why not?  Is server.gold used anywhere else?

Reply:
Added back server content checks.